cli/cmd/generate_aws.go

```diff
@@ -0,0 +1,375 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/lwgenerate/aws"
+	"github.com/pkg/errors"
+)
+
+var (
+	// Define question text here so they can be reused in testing
+	QuestionEnableConfig                = "Enable Config Integration?"
+	QuestionEnableCloudtrail            = "Enable Cloudtrail Integration?"
+	QuestionAwsRegion                   = "Specify the AWS region Cloudtrail, SNS, and S3 resources should use:"
+	QuestionConsolidatedCloudtrail      = "Use consolidated Cloudtrail?"
+	QuestionUseExistingCloudtrail       = "Would you like to use an existing Cloudtrail?"
```

Comment:
Another nitpick: IMO it would be good to be consistent with the wording here, as we are fairly consistent throughout the rest of the questions. 
Either both to start with `Use` or `Would you like to use`

Reply:
Good catch @rmoles